singularity-eos/base/eos_error.hpp

```diff
@@ -19,7 +19,7 @@
 #include <stdexcept>
 #define EOS_ERROR(x) (throw std::runtime_error(x))
 #else
-#define EOS_ERROR(x) printf("%s\n", x); exit(1);
+#define EOS_ERROR(x) printf("%s\n", x); assert(false);
```

Comment:
:+1:

Reply:
> > While making changes to the test: would it make sense to make matid an input parameter (int matId = std::atoi(argv[2]); std::vector matids{matId};)?
> 
> Originally, I intended it to be possible to run the script on multiple matids in one call. But I think you're right, it's better to just make `matid` an input parameter and run the script multiple times if that's what is desired.

Just for clarity @Yurlungur, what code change is being requested and should I add it in this PR?

Reply:
This task will be done in a different PR.